windows/sound.cpp

```diff
@@ -54,7 +54,7 @@ QString CSound::LoadAndInitializeDriver ( QString strDriverName, bool bOpenDrive
     // if the selected driver was not found, return an error message
     if ( iDriverIdx == INVALID_INDEX )
     {
-        return tr ( "The current selected audio device is no longer present in the system." );
+        return tr ( "The selected audio device is no longer present in the system. Please check your audio device." );

```

Comment:
```suggestion
        return tr ( "The selected sound hardware is no longer present in the system. Please check your audio device." );

```

Reply:
To be discussed. Only open question on this PR. Therefore waiting for a concensus.